content/docs/command-reference/plots/index.md

```diff
@@ -37,18 +37,25 @@ This type of metrics files are created by users, or generated by user data
 processing code, and can be defined in `dvc.yaml` (`plots` field) for tracking
 (optional).
 
-DVC generates plots as HTML files that can be open with a web browser. These
-HTML files use [Vega-Lite](https://vega.github.io/vega-lite/). Vega-Lite is a
-declarative grammar for defining plots using JSON. The plots can also be saved
-as SVG or PNG image filed from the browser.
+DVC can work with two types of plots files:
+1. Data series files, which can be JSON, YAML, CSV or TSV.
+2. Image files, like JPEG, GIF, or PNG.
```

Comment:
> like JPEG, GIF, or PNG.

What other ones? Any one that the browser will render I assume?

Reply:
Not quite, as we recognize the images by extension.
I need to manually expand that list on the DVC side.
Created iterative/dvc#6714